### PR TITLE
Added signatureses limitations to signatures.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2020

Reference:
https://docs.microsoft.com/microsoft-365/admin/setup/create-signatures-and-disclaimers#limitations-of-organization-wide-signatures

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/policy-and-compliance/mail-flow-rules/signatures.md [hunk 1]
```diff
@@ -110,6 +110,18 @@ Here are some examples of the conditions and exceptions you can use.
 
 For a complete list of conditions and exceptions that you can use to target the disclaimer, see [Mail flow rule conditions and exceptions (predicates) in Exchange Server](conditions-and-exceptions.md).
 
+## Limitations of organization wide signatures
+
+You can't do the following with Exchange Server signatures:
+  
+- Insert the signature directly under the latest email reply or forward
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/policy-and-compliance/mail-flow-rules/signatures.md:117]:
```suggestion
- Insert the signature directly under the latest email reply or forward.
```

File: Exchange/ExchangeServer/policy-and-compliance/mail-flow-rules/signatures.md [hunk 2]
```diff
@@ -110,6 +110,18 @@ Here are some examples of the conditions and exceptions you can use.
...
+    
+- Display server-side email signatures in users' Sent Items folders
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/policy-and-compliance/mail-flow-rules/signatures.md:119]:
```suggestion
- Display server-side email signatures in users' Sent Items folders.
```

File: Exchange/ExchangeServer/policy-and-compliance/mail-flow-rules/signatures.md [hunk 3]
```diff
@@ -110,6 +110,18 @@ Here are some examples of the conditions and exceptions you can use.
...
+    
+- Skip lines which contain variables that couldn't be updated (e.g. because the value wasn't provided for a user)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/policy-and-compliance/mail-flow-rules/signatures.md:121]:
```suggestion
- Skip lines which contain variables that couldn't be updated (for example if the value wasn't provided for a user).
```

**Comment by scanum** [comment]
@mattpennathe3rd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**